lib/manageiq/rpm_build/setup_source_repos.rb

```diff
@@ -5,13 +5,13 @@
 module ManageIQ
   module RPMBuild
     class SetupSourceRepos
-      attr_reader :git_tag, :github_url
+      attr_reader :git_ref, :github_url
 
-      def initialize
+      def initialize(ref)
         puts "\n---> #{self.class.name}::#{__method__}"
         options = YAML.load_file(CONFIG_DIR.join("options.yml"))
```

Comment:
Maybe this could be moved to a top-level ConfigOptions class (or use the config gem)...then you can access those options from the bin script.

Reply:
nvm, since you are going to drop the check it's not needed for this PR(though maybe that could be a separate refactoring).